hw/drivers/led/lp5523/src/lp5523.c

```diff
@@ -658,15 +669,18 @@ lp5523_verify_pr_instruction(struct led_itf *itf, uint8_t addr,
     uint16_t *ins)
 {
     int rc;
-    uint8_t mem[2];
+    uint8_t byte1[1];
+    uint8_t byte2[1];
     uint16_t ver;
 
-    rc = lp5523_get_n_regs(itf, LP5523_PROGRAM_MEMORY + (addr << 1), mem, 2);
+    addr = addr << 1;
+    rc = lp5523_get_n_regs(itf, LP5523_PROGRAM_MEMORY + (addr), byte1, 1);
+    rc |= lp5523_get_n_regs(itf, LP5523_PROGRAM_MEMORY + (addr + 1), byte2, 1);
```

Comment:
I guess lp5525_get_n_regs() does not return bit mask so this shortcut could be omitted. Same for other places.